### PR TITLE
1_kala

### PR DESCRIPTION
**Pull request by whitecollin**
update: todo_app

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,37 +46,46 @@ This repository is designed to help newbies practice using Git and GitHub by cre
...
+
 20. The admin will review the PR and say **Congrats** if everything looks good.
 
 ## Contributors
+
 - [John Doe](./john-doe/README.md)
-- [Sebastian Harvey](./jsstar/README.md)
\ No newline at end of file
+- [Sebastian Harvey](./jsstar/README.md)
+- [White Collin](./whitecollin/)
```
Comment (oddcommits) [inline README.md:91]: add a new line at the end of file

File: whitecollin/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# About Me
+Hi! I'm White Collin. I love programming very much.
```
Comment (oddcommits) [inline whitecollin/README.md:2]: could you write more about you?

**Review by oddcommits — CHANGES_REQUESTED** [review]
much better! just 2 minor fixes!

File: whitecollin/README.md (hunk 2 above)
Comment (whitecollin) [reply whitecollin/README.md:2]: fixed it

File: README.md (hunk 1 above)
Comment (whitecollin) [reply README.md:91]: fixed it

**Review by oddcommits — APPROVED** [review]

**Pull request merged**